andela/odin-ah-frontend
# #160452423 setup an empty react project

#### What does this PR do?
Sets up an empty react app using create-react-app tool to generate the boilerplate
#### Description of Task to be completed?
- setup a react boilerplate using create react app
- use Airbnb linting option
- add pull request template
#### How should this be manually tested?
- Clone the repo 
- Checkout to this branch
-  Run the app using `npm run start`
- Go to localhost:3000 to see the running app 
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#160452423](https://www.pivotaltracker.com/story/show/160452423)
#### Screenshots (if appropriate)
#### Questions:
